src/app/switcher/switcher-program/switcher-program.component.ts

```diff
@@ -43,6 +42,10 @@ export class SwitcherProgramComponent extends RouterEnter {
       });
   }
 
+  ngAfterContentChecked() {
+    document.getElementById('test-title').focus();
```

Comment:
Same as Html content. Can we use something like 'page-title' instead of 'test-title'.